mesh_layers/include/mesh_layers/border_layer.h

```diff
@@ -141,6 +141,9 @@ class BorderLayer : public mesh_map::AbstractLayer
     double border_cost = 1.0;
     double factor = 1.0;
   } config_;
+
+  // put this to base class?
+  std::string name;
```

Comment:
Wait, is this even used? I think you're only using the `layer_name_` attribute from the base class in the cpp file.

Looks like this is a leftover from experimenting.

Reply:
Yes, it's basically how it is written in the comments. "Put this to base class" then I jumped to the base class and saw, there is a variable already. So it can safely be removed.